tools/rule-types.json

```diff
@@ -88,6 +88,7 @@
     "no-console": "suggestion",
     "no-const-assign": "problem",
     "no-constant-condition": "problem",
+    "no-constructor-return": "suggestion",
```

Comment:
should be "problem"?

Reply:
Why?

Reply:
returning a value in constructor is mostly a mistake, as described in the OP.

Reply:
@golopot What's your opinion?

Reply:
"Problem" seems more appropriate.